control-plane-operator/controllers/hostedcontrolplane/konnectivity/params.go

```diff
@@ -31,6 +33,22 @@ func NewKonnectivityParams(hcp *hyperv1.HostedControlPlane, images map[string]st
 		ExternalPort:            externalPort,
 		OwnerRef:                config.OwnerRefFrom(hcp),
 	}
+	p.ServerDeploymentConfig.LivenessProbes = config.LivenessProbes{
+		konnectivityServerContainer().Name: {
+			Handler: corev1.Handler{
+				HTTPGet: &corev1.HTTPGetAction{
+					Scheme: corev1.URISchemeHTTP,
+					Port:   intstr.FromInt(int(healthPort)),
+					Path:   "healthz",
+				},
+			},
+			InitialDelaySeconds: 300,
```

Comment:
what makes this livenessProbe values being different than the ones for the kas?

```
			InitialDelaySeconds: 300,
			PeriodSeconds:       180,
			TimeoutSeconds:      160,
			FailureThreshold:    6,
			SuccessThreshold:    1,
```

i.e the PeriodSeconds and the FailureThreshold?

Reply:
this is actually a good point I should tailor these more

Reply:
adjuseted